src/transformers/models/deit/modeling_deit.py

```diff
@@ -477,7 +477,6 @@ class PreTrainedModel
     def forward(
         self,
         pixel_values=None,
-        attention_mask=None,
```

Comment:
let's leave it for now - we can clean it up once the generate refactor is done: https://github.com/huggingface/transformers/issues/14421